code/modules/mob/living/silicon/ai/latejoin.dm

```diff
@@ -13,19 +13,9 @@ GLOBAL_LIST_EMPTY(empty_playable_ai_cores)
 	// We warned you.
 	GLOB.empty_playable_ai_cores += new /obj/structure/AIcore/deactivated(loc)
 	GLOB.global_announcer.autosay("[src] has been moved to intelligence storage.", "Artificial Intelligence Oversight")
-
-	//Handle job slot/tater cleanup.
-	var/job = mind.assigned_role
-
-	SSjobs.FreeRole(job)
-
-	if(mind.objectives.len)
-		mind.objectives.Cut()
-		mind.special_role = null
-	else
-		if(SSticker.mode.name == "AutoTraitor")
-			var/datum/game_mode/traitor/autotraitor/current_mode = SSticker.mode
-			current_mode.possible_traitors.Remove(src)
+	control_disabled = TRUE // Ensure the AI is not chosen again as target
```

Comment:
I might be missing something, but as far as I can tell this variable is never actually referenced in objective code?

Reply:
It is not used there indeed. But it is used in `active_ais`. I'll update the comment